nce/NCE-Pairwise-SM/README.md

```diff
@@ -0,0 +1,88 @@
+## NCE-SM model
+
+#### References:
++ Aliaksei _S_everyn and Alessandro _M_oschitti. 2015. Learning to Rank Short Text Pairs with Convolutional Deep Neural
+Networks. In Proceedings of the 38th International ACM SIGIR Conference on Research and Development in Information 
+Retrieval (SIGIR '15). ACM, New York, NY, USA, 373-382. DOI: http://dx.doi.org/10.1145/2766462.2767738
+
++ Jinfeng Rao, Hua He, and Jimmy Lin. [Noise-Contrastive Estimation for Answer Selection with Deep Neural Networks.](http://dl.acm.org/citation.cfm?id=2983872) *Proceedings of the 25th ACM International on Conference on Information and Knowledge Management (CIKM 2016)*, pages 1913-1916.
+
+### Requirements
+```
+nltk==3.2.2
+numpy==1.11.3
+pytorch==0.1.12
+```
+
+The code uses torchtext for text processing. Set torchtext:
```

Comment:
Update these instructions to install using `pip` from `PyPI` instead of installing from source.